python/ml4ir/features/feature_layer.py

```diff
@@ -43,11 +28,12 @@ def get_sequence_encoding(input, feature_info, max_num_records):
     if feature_info["tfrecord_type"] == TFRecordTypeKey.CONTEXT:
         # If feature is a context feature then tile it for all records
         encoding = tf.expand_dims(encoding, axis=1)
-        encoding = K.repeat_elements(encoding, rep=max_num_records, axis=1)
-
+        # encoding = tf.tile(encoding,
```

Comment:
commented out?

Reply:
Yeah, I moved it out of this method. Look below.

```
tile_dims = tf.shape(
                            tf.expand_dims(
                                tf.expand_dims(tf.gather(inputs["mask"], indices=0), axis=0),
                                axis=-1,
                            )
                        )
                        encoding = tf.tile(encoding, tile_dims)
```

I will remove this commented out code.